python/packaging.bzl

```
@@ -310,6 +319,9 @@ _other_attrs = {
         ),
         default = "",
     ),
+    "stamp": attr.bool(
+        doc = "Enable stamping of the version attribute",
```

Can you add an example of what this would resolve?
```suggestion
        doc = "Enable stamping of the version attribute. Stamp variables should be wrapped in brackets. Eg: `1.2.3-{BUILD_HOST}`",
```
Something like that